svc-bgs-api/src/integrationTest/add_note_tests.rb

```diff
@@ -0,0 +1,35 @@
+# Inspired by the README at https://github.com/ruby-amqp/bunny
+
+require 'bunny'
+require 'json'
+
+require_relative '../config/constants'
+
+# Constants used by integration tests
+ADD_NOTE_REPLY_QUEUE = "add-note-response"
+ADD_NOTE_PAYLOAD = "{\\\"veteranNote\\\":\\\"test\\\",\\\"veteranParticipantId\\\":111}"
+CORRELATION_ID = "1234"
+
+
+def test_successful_add_note(ch)
```

Comment:
We should plan on using a proper testing lib in ruby such as RSpec // Minitest etc

Reply:
yeah it looks like right now, we'd have to change our application model to be more module based in order to really leverage these tools, and I didn't want to make those changes as part of just getting our integration tests out of GH actions.